### PR TITLE
RONDB-675: Build & publish RonDB for ARM

### PR DESCRIPTION
**Pull request by olapiv**
https://hopsworks.atlassian.net/browse/RONDB-675

### CODE REVIEW

File: .github/workflows/publish_rondb.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Set up SSH
+        run: |
+          mkdir -p ~/.ssh
+          echo "$SSH_PRIVATE_KEY" > ~/.ssh/id_rsa
+          chmod 600 ~/.ssh/id_rsa
+          ssh-keyscan -H $REPO_HOPSWORKS_HOSTNAME >> ~/.ssh/known_hosts
```
Comment (svenssonaxel) [inline .github/workflows/publish_rondb.yml]: Can we get this from any other source? E.g. hard-code it like the other vars above?

File: .github/workflows/publish_rondb.yml [hunk 2]
```diff
@@ -0,0 +1,71 @@
+name: Publish RonDB Tarball
+on:
+  push:
+    branches:
```
Comment (svenssonaxel) [inline .github/workflows/publish_rondb.yml:4]: Shouldn't this happen only on tags?

File: .github/workflows/publish_rondb.yml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Set up SSH
+        run: |
+          mkdir -p ~/.ssh
+          echo "$SSH_PRIVATE_KEY" > ~/.ssh/id_rsa
```
Comment (svenssonaxel) [inline .github/workflows/publish_rondb.yml]: `id_ed25519`

File: .github/workflows/publish_rondb.yml (hunk 2 above)
Comment (olapiv) [reply .github/workflows/publish_rondb.yml:4]: We'll do that once we use tags

**Review by svenssonaxel — APPROVED** [review]

**Pull request merged**